modules/dynamic-provisioning-azure-disk-definition.adoc

```diff
@@ -15,29 +15,22 @@ metadata:
   name: slow
 provisioner: kubernetes.io/azure-disk
 parameters:
-  storageAccount: azure_storage_account_name  <1>
-  storageaccounttype: Standard_LRS  <2>
-  kind: Dedicated  <3>
+  storageAccount: azure_storage_account_name <1>
+  storageaccounttype: Standard_LRS <2>
+  kind: Managed <3>
```

Comment:
@bobfuru  How about using storageclass defined in https://github.com/openshift/cluster-storage-operator/blob/master/assets/storageclasses/azure.yaml?

Reply:
Love it. Done.